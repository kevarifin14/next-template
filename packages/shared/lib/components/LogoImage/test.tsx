import { render } from '@testing-library/react';
import React from 'react';

import { LogoImage } from '.';

describe('LogoImage', () => {
  const renderLogoImage = (props = {}) => (
    render(
      <LogoImage
        {...props}
      />,
    )
  );

  it('matches snapshot', () => {
    const { baseElement } = renderLogoImage();
    expect(baseElement).toMatchSnapshot();
  });
});
