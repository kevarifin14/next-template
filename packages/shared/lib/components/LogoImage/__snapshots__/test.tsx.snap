// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoImage matches snapshot 1`] = `
<body>
  <div>
    <img
      alt="logo"
      class="mx-auto w-auto"
      src="/logo.png"
    />
  </div>
</body>
`;
